src/api-reference/direct-connects/hotel-service-4/v4.endpoints.md

```diff
@@ -1797,7 +1797,7 @@ Information about Point of Sale (POS), traveler, and user associated with the re
 |Name|Type|Format|Description|
 |---|---|---|---|
 `confirmationCodes`|[`ConfirmationCode`](#schemaconfirmationcode)|-|**Required**|
-`status`|[`ReservationStatus`](#schemareservationstatus)|-|**Required** Supported values: `PENDING_CONFIRMATION`, `RESERVED`, `CANCELLED`|
+`status`|[`ReservationStatus`](#schemareservationstatus)|-|**Required** Supported values: `PENDING_CONFIRMATION`, `RESERVED`, `CANCELLED`. Please note that `PENDING_CONFIRMATION` is not currently supported and reserved for future use.|
```

Comment:
@kunalshah01 We shouldn't list a supported value that's not actually supported. Please remove `PENDING_CONFIRMATION` from the list of values and the note.

Reply:
Removed